examples/style-swin-gan/styleswin_tc.ipynb

```diff
@@ -449,6 +449,10 @@
    ],
    "source": [
     "# load trained weights from plane Trans-thalamus\n",
+    "\n",
+    "# Hi Sean\n",
+    "# Yes, it would be great to have checkpoints integrated at leasts in the notebooks\n",
+    "# Hence we can run the following lines here. Thanks, --Miguel\n",
     "discriminator.load_state_dict(torch.load('data/saved_tt_256/d_500.pt'))\n",
     "generator.load_state_dict(torch.load('data/saved_tt_256/g_500.pt'))\n"
```

Comment:
Hi @seansunn , can you please help with this one?

Reply:
Hi @mxochicale , the checkpoint is quite large (>100Mb); I'll try to upload via git lfs.

Reply:
Hi @seansunn 
It would be great to have those checkpoints less than 100Mb because of the github data quote https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github.

* How many checkpoints you have and whats the total size? 
* Do you know good practices on how and where to save those checkpoints? 
I think we can potentially use [zenodo ](https://zenodo.org/) to host our checkpoints!  Thanks, --Miguel

Reply:
Hi @mxochicale , I've re-trained the models (one on TT plane and the other on TC plane), and I have ~165Mb for the Generator and ~103Mb for the Discriminator for each model.

Therefore I have a total of 4 checkpoints, and the total size is ~530Mb. I've seen people using google drive or dropbox for sharing checkpoints; zenodo is also great!